### ISSUE TITLE
Support for TVEventHandler of react-native is missing in Typescript

### ISSUE THREAD

**Issue by SenthilKumar-J** [microsoft/TypeScript#35931]
**TypeScript Version:**  2.9

**Search Terms:** TVEventHandler

**Code**
```ts
import { TVEventHandler } from "react-native";
```
Also tried to import using, 
```ts
// tslint:disable-next-line:no-var-requires
// let TVEventHandler = require("TVEventHandler");
```

**Expected behavior:**
Typescript should compile and execute properly. 
**Actual behavior:**
Module '"react-native"' has no exported member 'TVEventHandler' during compilation.
Support for TVEventHandler is not available even in the latest version, 3.7

**Playground Link:** 

**Related Issues:** No

**Comment by j-oliveras** [comment]
Probably you should report this to https://github.com/DefinitelyTyped/DefinitelyTyped repository because it have the type definitions for `react-native`.

**Comment by j-oliveras** [comment]
Duplicate of https://github.com/DefinitelyTyped/DefinitelyTyped/issues/41229
